applications/FSIapplication/python_scripts/fsi_coupling_interface.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.FSIApplication as KratosFSI
+
+class FSICouplingInterface():
+
+    def __ValidateSettings(self, settings):
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "model_part_name": "",
+            "father_model_part_name": "",
```

Comment:
```suggestion
            "parent_model_part_name": "",
```

please don't specify the gender xD

Joke aside, this is more coherent with the naming in ModelPart